blockscout-ens/bens-server/src/settings.rs

```diff
@@ -28,10 +28,49 @@ impl ConfigSettings for Settings {
     const SERVICE_NAME: &'static str = "BENS";
 }
 
+// TODO: move database settings to blockscout-service-launcher
 #[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
 #[serde(deny_unknown_fields)]
 pub struct DatabaseSettings {
-    pub url: String,
+    pub connect: DatabaseConnectSettings,
+}
+
+#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
+#[serde(deny_unknown_fields, rename_all = "lowercase")]
+pub enum DatabaseConnectSettings {
+    Url(String),
+    KV(DatabaseKVConnection),
```

Comment:
We've tried to correspond to this styleguides till the moment which suggests to count acronyms as one word - https://rust-lang.github.io/api-guidelines/naming.html
```suggestion
    Kv(DatabaseKvConnection),
```